prep-css-flexbox/justify-content.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+#card-deck {
+  display: flex;
+  flex-basis: 25%;
```

Comment:
Hi Jayden, the `flex-basis` property should be used on "flex items", that is, elements who are children of an element who has `display: flex`. So you want to move your `flex-basis` to the `.card-container` rule-set.